docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/03_LORIS_MRI_Install/Signal_Viewer_UI_Install.md

```diff
@@ -0,0 +1,277 @@
+
+
+# Signal Viewer UI Install
+
+> Last update: 2022-11-02
+
+## Context
+
+This document is a guide to install and configure Loris HBCD to use the `Signal Viewer` UI for EEG (BIDS files).
```

Comment:
This document is a guide to install and configure Loris to use the EEG Visualization features.

-- 
It's not the `Signal Viewer` alone, but also the Electrode map and future tools to come.